benches/bench.rs

```diff
@@ -138,12 +161,10 @@ fn bench_with_input_sizes(
 
     for input_size in input_sizes.iter() {
         if use_marf {
-            let headers_db = SimHeadersDB::new();
-
-            let mut marf = setup_chain_state(MARF_SCALE, &headers_db);
+            let mut marf = setup_chain_state(MARF_SCALE, &TestHeadersDB);
```

Comment:
I used the following two lines in several analysis bench functions: 
```
let headers_db = SimHeadersDB::new();
let mut marf = setup_chain_state(MARF_SCALE, &headers_db);
```

Should we switch those occurrences all to `let mut marf = setup_chain_state(MARF_SCALE, &TestHeadersDB);`?

Reply:
Yea, we should be using the `SimHeadersDB` here, however I've been having some issues with it so I switched back to `TestHeadersDB`. Will fix this and attempt to sort out the issues with our `SimHeadersDB` usage